Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/asset/AssetType.java

```diff
@@ -0,0 +1,39 @@
+package pg.autyzm.friendly_plans.asset;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.io.FilenameUtils;
+
+public final class AssetType {
+
+    public static final String PICTURE = "PICTURE";
+    public static final String SOUND = "SOUND";
```

Comment:
I think it's good idea. The only problem is that greenDao doesn't accept enum type but I'll implement helper toString method - it's better than validation + exception in dao